app/assets/javascripts/answers.js

```diff
@@ -36,21 +38,25 @@ enableSubmitAnswer();
 // runs each time the user presses a key in the #answer_text form field
 $('#answer_text').keyup(enableSubmitAnswer); 
 
-
-  $("#submit_answer").click(function(){
+// When a user submits an answer. The real answer and response form fade in. 
+  $("#submit_answer").click(function(ev){
+      // prevents the program from automatically adding a hidden attribute to the "respnse" so that it can be resused in later modals
```

Comment:
How about we update this to call out the issue with bootstrap + jquery working together? Something like this:
```
// prevents bootstrap from automatically adding a hidden attribute; we are explicitly hiding and showing using jquery instead
```